server/status/runtime_jemalloc.go

```diff
@@ -19,6 +19,7 @@
 package status
 
 // #cgo darwin CPPFLAGS: -I../../../c-jemalloc/darwin_includes/internal/include
+// #cgo freebsd CPPFLAGS: -I../../../c-jemalloc/freebsd_includes/internal/include
```

Comment:
This is mildly frustrating: if you wanted to communicate that the two files did not use the same ordering, why didn't you write so in the first place? It would have saved us two backs-and-forth. Anyway, it's corrected now.